Doc/library/stdtypes.rst

```diff
@@ -1131,6 +1131,9 @@ Lists are mutable sequences, typically used to store collections of
 homogeneous items (where the precise degree of similarity will vary by
 application).
 
+Despite their name, lists are more akin to arrays in other languages than to
```

Comment:
“despite their name”  seems redundant.
“List” is datastructure independent, abstract word.
For example, C# uses IList and ArrayList for sequence (indexable). And LinkedList for linked list.